command/agent/config/test-fixtures/config-cache.hcl

```diff
@@ -21,6 +21,15 @@ auto_auth {
 
 cache {
 	use_auto_auth_token = true
+	persist = {
+		path = "/tmp/bolt-file.db"
+		keep_after_import = true
+		exit_on_err = true
+		crypto = {
```

Comment:
Is this a bug? Should it be `crypto "kubernetes" {}` or are we testing for this?

Reply:
I believe this is the non-embedded way to specify the type, going off of these test fixtures:

embedded:
https://github.com/hashicorp/vault/blob/e0b9cf80a415b109cb87d917a35344118ab8a12b/command/agent/config/test-fixtures/config-embedded-type.hcl#L4-L10

not embedded:
https://github.com/hashicorp/vault/blob/e0b9cf80a415b109cb87d917a35344118ab8a12b/command/agent/config/test-fixtures/config.hcl#L4-L10